src/game/mario_actions_automatic.c

```diff
@@ -725,7 +725,9 @@ s32 act_in_cannon(struct MarioState *m) {
             marioObj->oMarioCannonInputYaw -= (s16)(m->controller->stickX * 10.0f);
 
             m->faceAngle[0] = CLAMP(m->faceAngle[0], 0, DEGREES(80));
-            marioObj->oMarioCannonInputYaw = CLAMP(marioObj->oMarioCannonInputYaw, -0x4000, 0x4000);
```

Comment:
Actually wait, I think this variable is an s32, so it actually needs to be clamped no matter what. This is actually good news for the bparam idea though.